lighthouse-core/audits/metrics/largest-contentful-paint.js

```diff
@@ -27,21 +27,32 @@ class LargestContentfulPaint extends Audit {
       title: str_(i18n.UIStrings.largestContentfulPaintMetric),
       description: str_(UIStrings.description),
       scoreDisplayMode: Audit.SCORING_MODES.NUMERIC,
-      requiredArtifacts: ['traces', 'devtoolsLogs'],
+      requiredArtifacts: ['traces', 'devtoolsLogs', 'TestedAsMobileDevice'],
     };
   }
 
   /**
-   * @return {LH.Audit.ScoreOptions}
+   * @return {{mobile: {scoring: LH.Audit.ScoreOptions}, desktop: {scoring: LH.Audit.ScoreOptions}}}
    */
   static get defaultOptions() {
     return {
-      // 25th and 13th percentiles HTTPArchive -> median and p10 points.
-      // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2020_02_01_mobile?pli=1
-      // https://web.dev/lcp/#what-is-a-good-lcp-score
-      // see https://www.desmos.com/calculator/1etesp32kt
-      p10: 2500,
-      median: 4000,
+      mobile: {
+        // 25th and 13th percentiles HTTPArchive -> median and p10 points.
+        // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2020_02_01_mobile?pli=1
+        // https://web.dev/lcp/#what-is-a-good-lcp-score
+        // see https://www.desmos.com/calculator/1etesp32kt
+        scoring: {
+          p10: 2500,
+          median: 4000,
+        },
+      },
+      desktop: {
+        // SELECT QUANTILES(lcp, 10), QUANTILES(lcp, 30) FROM [httparchive.pages.2020_04_01_desktop]
```

Comment:
did you update the query here @brendankenny? I'm a little confused.

AFAICT it's supposed to return every ~11th quantile and then every ~3.3rd quantile but which ones were used? also I get validation errors that `lcp` isn't a property in the `pages` table

shouldn't we use 25th and 5th if we can and stick to the `QUANTILES(lcp, 21)` query?

would also be nice to standard sql in our new queries moving forward and use `APPROX_QUANTILES` but no biggie that's a separate topic :)

Reply:
> AFAICT it's supposed to return every ~11th quantile and then every ~3.3rd quantile but which ones were used?

the HTTP Archive WPT desktop/mobile distributions are far enough away from the Lighthouse distributions that we can't really base it just on the desktop numbers so simply I don't think. As an example, the `pages_mobile` numbers would give `{p10: 4648, median: 5978}` for the mobile scoring control points here (vs our 2500/4000). That's why I mentioned comparing those curves to the Lighthouse curve, then I tried to pick analogous points to our mobile control points based on the `pages_desktop` curve.

> shouldn't we use 25th and 5th if we can and stick to the `QUANTILES(lcp, 21)` query?

But these are pretty close to my results (`{p10: 1193, median: 2419}`) so let's ignore what I thought was useful and let's keep it somewhat standardized :)

> would also be nice to standard sql in our new queries moving forward and use APPROX_QUANTILES but no biggie that's a separate topic :)

ha, I forgot that's actually the legacy SQL function and thought it was just a shorthand for `APPROX_QUANTILES` and an offset :) I can update to that.

> also I get validation errors that lcp isn't a property in the pages table

yeah, it's not in any of the summaries, this is going into the json payload and getting `"$['_chromeUserTiming.LargestContentfulPaint']"`. I can include that as well.

Reply:
> this is going into the json payload and getting "$['_chromeUserTiming.LargestContentfulPaint']". I can include that as well.

Gotcha, ya working queries would be 👌 (*must imagine Paul's signature noise while reading that*)